src/toast/scripts/toast_fake_focalplane.py

```diff
@@ -190,7 +190,7 @@ def main():
     # Guard against being called with multiple processes
     mpiworld, procs, rank = get_world()
     if rank == 0:
-        with H5File(args.out, "w", comm=world, force_serial=True) as f:
+        with H5File(args.out, "w", comm=mpiworld, force_serial=True) as f:
```

Comment:
How did this work before? I know I have run the fake focalplane script fairly recently.

Reply:
I'm not sure.  This change was made 7 months ago:  https://github.com/hpc4cmb/toast/pull/551/files and it is possible that I have not run that particular script since then.